template/circle.yml

```diff
@@ -6,7 +6,8 @@ machine:
 
 checkout:
   post:
-    - git fetch origin gh-pages --depth 10
+    - if [ -e .git/shallow ]; then git fetch origin --unshallow; fi
+    - git fetch origin gh-pages
```

Comment:
Do we really need the unshallow fetch?  as long as the gh-pages fetch is complete, then this should be fine, right?  I would just drop this entire step, since the makefile fetches gh-pages as needed now.